README.md

```diff
@@ -278,6 +288,10 @@ If you need to get a reference to a previously registered metric, you can use `r
 You can remove all metrics by calling `register.clear()`. You can also remove a single metric by calling
 `register.removeSingleMetric(*name of metric*)`.
 
+##### Cluster metrics
+
+You can get aggregated metrics for all worker processes in a cluster by running `register.clusterMetrics((err, metrics) => {...})`, which will be called with an error, or with the aggregated metrics string for prometheus to consume.
```

Comment:
I prefer Promise these days because of `async-await`, which is in node 8 which will be LTS in a couple of months. Makes for cleaner code.

That said, I don't feel strongly about it